requirements/edx-sandbox/base.txt

```diff
@@ -4,7 +4,6 @@
 #
 #    make upgrade
 #
-
```

Comment:
Hmm, I've been seeing these blank lines getting added and removed on subsequent PRs.  I think pip-compile adds it on Linux but omits it on the Mac; I'll file a bug for that and standardize it in the post-processing script in the meantime.